<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer_Window</class>
 <widget class="QWidget" name="Timer_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="description_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>ALLONTANARSI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timer_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                 color: rgb(255, 255, 255);
                 border-radius:10px;
             </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/cancel/res/cancel.png</normaloff>:/cancel/res/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="download_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                 color: rgb(255, 255, 255);
                 border-radius:10px;
             </string>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>Timer_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
